p4-16/psa/PSA.mdk

```diff
@@ -295,11 +293,15 @@ the contents of several metadata fields in the struct
             This affects any copies made below.
     }
     if (ostd.clone) {
-        create a copy of the packet to the clone target
-        // TBD: Need a way to specify one (or more?) among multiple clone
-        // targets, if more than one such target exists.  Also to specify
-        // if anything is different about the cloned packet vs. other
-        // copies that might be made below.
+        create a copy of the packet and send it to the packet buffer,
+            after which it will start egress processing with
+            egress_port equal to the same value as the packet that was
```

Comment:
egress_port (clone_port?) of the cloned packet can be set to any port, and should not be restricted to the egress_port of the original packet coming into the egress pipeline.

Reply:
ingress2egress clone allows cloning to CPU port.  Should egress2egress clone allow or prohibit this possibility?

Reply:
It should allow this possibility.

Reply:
Done.